src/System.Net.Http/src/uap/System/Net/HttpClientHandler.cs

```diff
@@ -350,36 +356,51 @@ public SslProtocols SslProtocols
 
         public HttpClientHandler()
         {
-            _rtFilter = new RTHttpBaseProtocolFilter();
-            _handlerToFilter = new HttpHandlerToFilter(_rtFilter);
+            _rtFilter = CreateFilter();
+
+            _handlerToFilter = new HttpHandlerToFilter(_rtFilter, this);
             _handlerToFilter.RequestMessageLookupKey = RequestMessageLookupKey;
             _handlerToFilter.SavedExceptionDispatchInfoLookupKey = SavedExceptionDispatchInfoLookupKey;
             _diagnosticsPipeline = new DiagnosticsHandler(_handlerToFilter);
 
             _clientCertificateOptions = ClientCertificateOption.Manual;
 
+            _useCookies = true; // deal with cookies by default.
```

Comment:
Same comment as in the previous PR; it'd be better to use the values from https://github.com/dotnet/corefx/blob/master/src/Common/src/System/Net/Http/HttpHandlerDefaults.cs (and now that there's a shared partial handler class at https://github.com/dotnet/corefx/blob/master/src/System.Net.Http/src/System/Net/Http/HttpClientHandler.cs, these consts could just be moved there rather than having a separate file)